nbdev/sync.py

```diff
@@ -8,7 +8,9 @@
 from .config import *
```

Comment:
I tested this by running `nbdev_update` in the nbdev repo and ensuring that it didn't change any of our notebooks

Reply:
Good idea